src/containers/Pomodoro/Pomodoro.js

```diff
@@ -185,10 +190,16 @@ class Pomodoro extends Component {
           cancelSettings={this.handleCancel}
         />
       </TimerBox>
-    ) : (
-      <TimerBox>
-        <h2>{isBreak && sessionCount % 4 === 0 && 'LONG'}</h2>
-        <h2>{isBreak ? 'BREAK' : 'SESSION'}</h2>
+      )
+    } else if (sessionLogOn && !settingsOn) {
+      currentPage = (
+          <SessionLog returnToTimer={this.handleReturnFromSessionLog} />
+      )
+    } else {
+      currentPage = (
```

Comment:
Extract this whole thing to a dumb component `Timer`? that just takes the needed props.